### PR TITLE
VEHICLE LIVE RISK PREDICTION

### PR DESCRIPTION
**Pull request by CoderOMaster**
# Pull Request for ML-Crate 💡

## Issue Title: VEHICLE LIVE RISK PREDICTION

- **Info about the related issue (Aim of the project)** : 
- **Name:** <Keshav Arora
- **Idenitfy yourself:** JWOC 2024

Closes: #issue number 473 that will be closed through this PR

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Put all the EDA results in the Images folder.
2. Can you implement Lasso, Ridge, XgBoost, Gradient Boosting and MLP in this dataset? That'll be a better comparison and model implementation showcase.

@CoderOMaster

**Comment by CoderOMaster** [comment]
Hi @abhisheks008 
I will put all eda images and also show one more algorithm since as described in detail that dataset is too perfect for any model to handle.I have reached this conclusion on extensive filtering.
Thank You

**Comment by CoderOMaster** [comment]
@abhisheks008 I have implemented all and xgboost in another notebook separately due to some detection issue in garuda linux.All images are added too for eda.I would like to request you that please consider this as hard problem since it took lot of analysis to conclude.

File: Vehicle Live Risk Prediction/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+
+
+
+
+
+These sample images show relationship of columns with each other(more detail in notebook 1 with EDA)
+
+**MODELS USED**
```
Comment (abhisheks008) [inline Vehicle Live Risk Prediction/README.md:56]: Update the model names as per the recent upgrade.

File: Vehicle Live Risk Prediction/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+4) Multivariate Logisitic Regression
+
+**LIBRARIES**
+1) PANDAS
+2) NUMPY
+3) TENSORFLOW
+4) MATPLOTLIB
+5) SEABORN
+
+**IMPLEMENTATION**
```
Comment (abhisheks008) [inline Vehicle Live Risk Prediction/README.md:69]: Update this too.

File: Vehicle Live Risk Prediction/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+1) Loaded the dataset with 20 columns having 15000 entries.
+2) Implementated EDA with visualisation to check for relationships between data.
+3) Model was trained efficiently with different algorithms
+4) 50+ risk_score was considered as parameter for accident risk,whereas less than value signified no risk of accident.
+4) Different notebooks with different models were used for clear and concise display of information for mentor.
+
+**RESULT**
```
Comment (abhisheks008) [inline Vehicle Live Risk Prediction/README.md:76]: Update the results section too. It's better to create a table like this,
| Models implemented | Accuracy scores |
| :--: | :--: |
|Random Forest| 99% |
| ... | ... |
| ... | ... |

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Requested some changes related to documentation.
@CoderOMaster 

This will be marked as Hard label, don't worry, your efforts will get the value as well.

File: Vehicle Live Risk Prediction/README.md (hunk 3 above)
Comment (CoderOMaster) [reply Vehicle Live Risk Prediction/README.md:83]: done sir

**Review by abhisheks008 — APPROVED** [review]
Approved.
@CoderOMaster

**Comment by abhisheks008** [comment]
Please share your email ID for further communication.
@CoderOMaster

**Comment by CoderOMaster** [comment]
@abhisheks008 
e-mail: keshavarorasci@gmail.com
Thank You

**Pull request merged**